avail-intellij-plugin/src/main/kotlin/org/availlang/intellij/plugin/module/AvailModuleWizardStep.kt

```diff
@@ -31,24 +31,69 @@
  */
 package org.availlang.intellij.plugin.module
 
+import avail.anvil.environment.GlobalEnvironmentSettingsV1
 import com.intellij.ide.util.projectWizard.ModuleWizardStep
+import com.intellij.openapi.module.Module
 import javax.swing.JComponent
-import javax.swing.JLabel
 
 /**
  * The [ModuleWizardStep] used to set up a new Avail project.
  *
  * @author Richard Arriaga
```

Comment:
Not gonna block this PR for this, but generally, if you work in a file someone else created, you should add your author tag below the other listed authors. So feel free to add your author tag here.